packages/clerk-js/src/ui/components/OrganizationProfile/ProfileSettingsPage.tsx

```diff
@@ -90,17 +89,16 @@ export const ProfileSettingsPage = withCardStateProvider(() => {
             onAvatarRemove={isDefaultImage(organization.imageUrl) ? null : onAvatarRemove}
           />
           <Form.ControlRow elementId={nameField.id}>
-            <Form.Control
+            <Form.PlainInput
               {...nameField.props}
               autoFocus
-              required
+              isRequired
             />
           </Form.ControlRow>
           <Form.ControlRow elementId={slugField.id}>
-            <Form.Control
+            <Form.PlainInput
```

Comment:
❓ This was required before, right?

Reply:
with the previous implementation of `FormControl` required would be overriden with required={isRequired} which in this situation is `false`. After checking the tests we had for ProfileSettings and the signature of `organization.update` i decided to remove this completely.